content/Helpers.js

```diff
@@ -340,7 +340,7 @@ Copper.parseLinkFormat = function(data) {
 	var links = new Object();
 	
 	// totally complicated but supports ',' and '\n' to separate links and ',' as well as '\"' within quoted strings
-	var format = data.match(/(<[^>]+>\s*(;\s*\w+\s*(=\s*(\w+|"([^"\\]*(\\.[^"\\]*)*)")\s*)?)*)/g);
+	var format = data.match(/(<[^>]+>\s*(;\s*\w+\s*(=\s*([^",;][^,;]*|"([^"\\]*(\\.[^"\\]*)*)")\s*)?)*)/g);

```

Comment:
The \w+ fulfills the function of "just a word" (including numbers). Your expression is too wide.